app/src/main/res/values-ca/strings.xml

```diff
@@ -52,8 +52,7 @@ i sempre saber on ets per no perdre on baixar de l\'autobús. ]]></string>
   <string name="action_fav_trip">Marcar trajecte com a favorit</string>
   <string name="action_unfav_trip">Desmarcar trajecte com a favorit</string>
   <string name="action_switch_locations">Intercanvia Des de i Cap a</string>
-  <string name="action_time_type">L\'hora és</string>
-  <string name="trip_dep">de sortida</string>
+	<string name="trip_dep">de sortida</string>
```

Comment:
Why do these use a different indentation? did you run tx pull --mode=developer?

Reply:
I noticed that too, it happened when deleting some now unused string resources via the Translation Editor in IntelliJ (s.th. the strings are automatically removed from all languages and Lint wouldn't complain about them). After this being merged to master, Transifex should pick the changes up and upon next `tx pull` the indentation should be consistent again.